### PR TITLE
[REEF-901] Add GetFileInfo/GetFileStatus API to IFileSystem

### PR DESCRIPTION
**Pull request by anupam128**
This addressed the issue by
 * Creating the new API
 * Adding trivial implementation to LocalFileSystem

JIRA:
 [REEF-901](https://issues.apache.org/jira/browse/REEF-901)

### CODE REVIEW

File: lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs [hunk 1]
```diff
@@ -87,6 +87,11 @@ public Uri CreateUriForPath(string path)
             return uri;
         }
 
+        public FileStatus GetFileStatus(Uri remoteFileUri)
```
Comment (afchung) [inline lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs]: Please add comment for public facing API.

File: lang/cs/Org.Apache.REEF.IO/FileSystem/Local/LocalFileSystem.cs [hunk 2]
```diff
@@ -48,6 +48,17 @@ public Uri CreateUriForPath(string path)
             return new Uri(path);
         }
 
+        public FileStatus GetFileStatus(Uri remoteFileUri)
```
Comment (afchung) [inline lang/cs/Org.Apache.REEF.IO/FileSystem/Local/LocalFileSystem.cs]: Could you add comments for this file as well? Thanks!

File: lang/cs/Org.Apache.REEF.IO/FileSystem/FileStatus.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+
+namespace Org.Apache.REEF.IO.FileSystem
+{
+    public class FileStatus
```
Comment (afchung) [inline lang/cs/Org.Apache.REEF.IO/FileSystem/FileStatus.cs]: sealed?

Comment (afchung) [inline lang/cs/Org.Apache.REEF.IO/FileSystem/FileStatus.cs]: Also, please add documentation.

**Comment by afchung** [comment]
@anupam128 This looks good, just needs some additional documentation. Thanks!

**Comment by anupam128** [comment]
Thanks @afchung! I have made the changes.

**Comment by afchung** [comment]
``RunSimpleEventHandlerOnLocalRuntime`` fails on my machine, but it is unrelated to your changes, since I've tested on a clean repository. Merged. Thanks @anupam128!

**Pull request closed without merging**